src/services/mod.rs

```diff
@@ -20,3 +22,12 @@ pub fn offset_page_meta(meta: &PageMetadata, offset: i64) -> String {
     }
     .to_url_string()
 }
+
+pub async fn backend_url(
```

Comment:
I would maybe name this something like `core_sevices_url`

Reply:
How about `core_uri` as you did in the example you sent me? It's a bit shorter which is nice.